tests/unit/components/BookingForm.spec.js

```diff
@@ -30,15 +34,15 @@ describe("Component BookingForm.vue", () => {
   });
 
   it("should render `DatePicker` component", () => {
-    expect(underTest.findComponent({ name: "v-date-picker" }).exists()).toBe(
+    expect(underTest.findComponent({ name: "bad-date-picker" }).exists()).toBe(
```

Comment:
This is fine, but you don't have to use the 'name', you can just do ...findComponent(BadDatePicker)....
It is up to you if you want to change it or not. I would change it just to be consistent how we use the method in the other tests.

Reply:
I will change it! It looks better to me ^^